util/execdetails/execdetails_test.go

```diff
@@ -138,6 +138,14 @@ func TestCopRuntimeStats(t *testing.T) {
 	if stats.ExistsRootStats(tableReaderID) == false {
 		t.Fatal("table_reader not exists")
 	}
+
+	cop.copDetails.ProcessedKeys = 0
+	cop.copDetails.RocksdbKeySkippedCount = 0
+	cop.copDetails.RocksdbBlockReadCount = 0
+	if cop.String() != "tikv_task:{proc max:1.000000001s, min:2ns, p80:1.000000001s, p95:1.000000001s, iters:4, tasks:2}, "+
```

Comment:
Can it be 1.001s? Looks to be not necessary to be this precise.

Reply:
Yes, maybe we can use a better way to print time.

Reply:
How about using `Microsecond` precision? @SunRunAway PTAL

Reply:
I have a proposal, for unit and precision for time and size:

1. unit rule: when readable value in larger unit is <10, use smaller unit.

e.g. 9ms -> 9000us, but 10ms is kept as 10ms

2. precision rule: when readable value >=10, use 0 decimals. when unit is the smallest (ns, byte), use 0 decimals. Otherwise, use 2 decimals.

e.g. 9.412345ms -> 9.41ms, 10.412345ms -> 10ms, 5.989s -> 6.00s, 100.45us -> 100us, 5 bytes, 0.12345MiB -> 124KiB (due to rule 1, use KiB instead of MiB)

3. This only affects display. Value should be stored using max precision, in case of future change.

What do you think?

Reply:
Estimated rows can be rounded to 1 dicimals while displaying I think, e.g. 4.3. 5.0.

Reply:
Agree with the time precision.
For estimated rows, I'm not sure, @SunRunAway @qw4990 What do you think?

Reply:
> Agree with the time precision. But I think always keep 2 decimals may be better. Such as the parent is 10.4 s, the child was 10.01 s?
> For estimated rows, I'm not sure, @SunRunAway @qw4990 What do you think?

Good point. I updated my proposal. For >= 10 values, 1 digit is reversed.

Reply:
For estRows, suggest always keeping 3 digits.

Reply:
The `estRows` was already been always keep 2 digital before.

![image](https://user-images.githubusercontent.com/26020263/99614636-e014a180-2a54-11eb-9d32-fb16eaf99296.png)